backend/src/main/java/ca/bc/gov/backendstartapi/entity/embeddable/AuditInformation.java

```diff
@@ -0,0 +1,41 @@
+package ca.bc.gov.backendstartapi.entity.embeddable;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Embeddable;
+import java.io.Serializable;
+import java.time.LocalDateTime;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+
+/** Used to track the changes made in a registry. */
+@Embeddable
+@NoArgsConstructor
+@RequiredArgsConstructor
+@Getter
+@Setter
+public class AuditInformation implements Serializable {
+
+  /** User that entered the registry in the system. */
+  @NonNull
+  @Column(name = "entry_userid", length = 30, nullable = false)
+  private String entryUserId;
+
+  @NonNull
+  @Column(name = "entry_timestamp", nullable = false)
+  private LocalDateTime entryTimestamp;
+
+  /** User who last updated this registry. */
+  @NonNull
+  @Column(name = "update_userid", length = 30, nullable = false)
+  private String updateUserId;
+
+  @NonNull
+  @Column(name = "update_timestamp", nullable = false)
+  private LocalDateTime updateTimestamp;
```

Comment:
@annibalsilva yes they can. I will change the table creation script so that both columns have default values for insert and update actions. I will also try to do the same for revision_count.
It is essential to say that the backend should send only Null values for those columns since the database will manage that data.

Reply:
@annibalsilva In Postgres, we can only have default values for inserts, not for updates. So we won´t be able to control the changes in those columns in the database. That management needs to be done in the backend.

Reply:
> That management needs to be done in the backend.

I'll put those in place, then.